### PR TITLE
[Issue 243] Remove "urth" references from HTML & CSS

### PR DESCRIPTION
**Pull request by dalogsdon**
Part of #243

### CODE REVIEW

File: jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-actions.js [hunk 1]
```diff
@@ -22,6 +22,8 @@ define([
 ) {
     'use strict';
 
+    var NS = 'jupyter-dashboard';
+
```
Comment (jhpedemonte) [inline jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-actions.js]: I agree with statement from morning scrum: don't make this "configurable", just change instances like in other files.

**Comment by jhpedemonte** [comment]
Tested, works as expected. Fix the inline comment and good to merge.

File: jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-actions.js (hunk 1 above)
Comment (dalogsdon) [inline jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-actions.js]: This was a remnant from the configurable value that I removed. I thought it still made sense as a constant value, but I will copy it back into the string values.

**Comment by dalogsdon** [comment]
Updated based on line comments.

**Pull request merged**